system7-tests/integration/case28-addDoesntAllowToAddSubrepoWithForbiddenTransportProtocolInSystemOptions.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+S7_SYSTEM_OPTIONS_PATH="/etc/s7options"
```

Comment:
This test requires 'root' privileges to be run, because we access /etc directory. Thus now test.sh should be run using sudo.